autosubmit/job/job_list.py

```diff
@@ -1487,11 +1481,11 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
                 dependencies_of_that_section.append(key_aux_stripped)
             # Adds the dependencies to the job, and if not possible, adds the job to the
             # problematic_dependencies
+            ## Adds the dependencies to the job, and if not possible, adds the job to the problematic_dependencies
             problematic_dependencies = self._calculate_natural_dependencies(dic_jobs, job,
-                dependency, date, member, chunk, graph, dependencies_keys_without_special_chars,
-                distances_of_current_section, distances_of_current_section_member, key,
-                dependencies_of_that_section, chunk_list, date_list, member_list,
-                special_dependencies, max_distance, problematic_dependencies)
+                        dependency, date, member, chunk, graph, distances_of_current_section, key,
+                        dependencies_of_that_section, chunk_list, date_list, member_list,
+                        special_dependencies, max_distance, problematic_dependencies)
```

Comment:
Same warning here about overly indentation. If it's ruff or pylint, we will need to prepare these changes later telling others to adjust their IDEs and use the linter :+1: